2024 rse call (#766)
RSE-Sheffield/RSE-Sheffield.github.io: pages/collaboration/RSEtime_call2024/call2024.md

---
title: Call for Proposal - RSE collaboration
permalink: /collaboration/RSEtime_call2024/
slug: index
type: text
---

### Rationale

#### Introduction

> The University of Sheffield recognizes the critical role that Research Software Engineers (RSEs) play in advancing scientific research. The University of Sheffield Research Software Engineering (RSE) team was founded in 2016 by two EPSRC RSE fellows to develop research software and collaborate with academics/researchers developing research software. The team has since grown to 11 members and has worked on a variety of projects with all faculties at the University. Areas of expertise within the group include: code optimisation and performance, reproducibility and embedding good software engineering practice within a project or team, developing/maintaining infrastructure, data management, developing computational workflows, GPU computing and Deep Learning, data science/visualisation, High Performance Computing using local/national/international/cloud systems, general software development, software/hardware interface, development of impact case studies, consultancy, training and education delivery and support.

Reviewer: > The team has since grown to 11 members and has worked on a variety of projects with all faculties at the University.

Possibly worth extending this to highlight that we've worked on small internally funded projects, large international consortiums (EU) and industrial. To highlight a different dimension of our breadth.

> To support the University research community we are pleased to announce a **call for proposals which will be supported by free at point-of-use RSE collaboration**. This initiative aims to facilitate access to RSE expertise for researchers who require assistance with software development, optimization, or other computational challenges.

Reviewer: > **call for proposals which will be supported by free at point-of-use RSE collaboration**.

This bit reads a bit weird. I don't think we should call an effective internal grant 'free at point of use' it's not a widely available free service. 

Maybe just name it '2024 RSE collaboration grant's call for proposals.' After which you can go on to explain it.

#### Objective
The objective of this call is to offer dedicated RSE collaboration to researchers who may lack the resources or expertise to effectively develop and maintain research software. By providing free access to RSE expertise, we aim to enhance the quality, reliability, and impact of research projects across diverse disciplines.

> #### Eligibility

Reviewer: Are you not going to explicitly say ML projects were covered by the prev CMI call, so are not eligible for this grant.

Author: I believe we should not restrict make constraints like this. If we have projects that are more suitable to the CMI-RSE call we could tell them no for this call but that we can take that on other scheme, under the CMI umbrella.

-Any research active staff, including PDRAs, and PhD students, affiliated to the University of Sheffield can apply to this call.
-Projects must involve significant software development that would benefit from RSE expertise.
-Preference may be given to projects with potential high impact or those addressing pressing scientific challenges.

#### Funding and support

> For this first call, the team will select two projects.**Selected proposals will receive 6 months @ 45-50% FTE of RSE support**. The start date will be decided by the RSE team based on the project timeline and RSEs availability. The duration of the project will not exceed 6 months but might be shortened with a higher FTE fraction (e.g. 3 months @ 100%). In addition, the collaboration will be continuous will not be made a shorter disjoint shorter periods of time.

Reviewer 2: Final sentence needs another look

Author: I accepted Neil's suggestions for this.

> For this first call, the team will select two projects.**Selected proposals will receive 6 months @ 45-50% FTE of RSE support**. The start date will be decided by the RSE team based on the project timeline and RSEs availability. The duration of the project will not exceed 6 months but might be shortened with a higher FTE fraction (e.g. 3 months @ 100%). In addition, the collaboration will be continuous will not be made a shorter disjoint shorter periods of time.

Reviewer 3: ```suggestion
For this first call, the team will select two projects.**Selected proposals will receive 6 months @ 45-50% FTE of RSE support**. The start date will be decided by the RSE team based on the project timeline and RSEs availability. The duration of the project will not exceed 6 months but might be shortened with a higher FTE fraction (e.g. 3 months @ 100%). In addition, the collaboration will be continuous and can not be split into multiple shorter, disjointed periods of time.
```

### Application guidelines

> An application template for this call has been prepared and is available [here](https://docs.google.com/document/d/1yzqS8gS-iCQ4HgM3dBcYEfYsS9E1Zm28CguZrTMl22M/edit?usp=sharing). We kindly request applicants to use this template for the pplication. This is a locked file so please make a copy to be able to complete it. We give more details about each section below.

Reviewer 2: missing 'a' on 'application'

Author: fixed

> -**Rational [max 600 words]:** Provide a brief description of the research project, its objectives and the role of software or computation methods in achieving these objectives.

Reviewer: ```suggestion
-**Abstract [max 600 words]:** Provide a brief description of the research project, its objectives and the role of software or computation methods in achieving these objectives.
```

Reviewer: Maybe also clarify the description should be accessible to reviewers outside their field.

> -**Rational [max 600 words]:** Provide a brief description of the research project, its objectives and the role of software or computation methods in achieving these objectives.

Reviewer 3: Need spaces after the bullet points for them to be correctly recognised as an unordered list in Markdown (obviously the other points below also need spaces!).

```suggestion
- **Rational [max 600 words]:** Provide a brief description of the research project, its objectives and the role of software or computation methods in achieving these objectives.
```

-**RSE support objectives [max 600 words]:** Outline the specific areas where RSE expertise is needed, such as software development, optimisation, debugging, use of HPC, etc… We remind you that, if selected, your project will receive 6 months half time. You will have to take into account project onboarding at the beginning of the collaboration and offboarding the project where we will handover the work done back to you.

-**Expected outcome [max 300 words]:** What does a successful collaboration look like to you? Describe the anticipated outcomes resulting from the contribution of RSE. 

> -**Project risks [max 200 words]:** What could make the collaboration go wrong? Describe briefly what are the potential risks of this project. (An obvious example: lack of data or no access to sensitive data)

Reviewer: ```suggestion
-**Project risks [max 200 words]:** What issues may cause the collaboration to be unsuccessful? Briefly describe the potential risks to the project. (A common example: lack of data or no access to sensitive data)
```
Lots I didn't like about this one.

> -**Impact statement [max 200 words]:** Explain how access to free RSE time would benefit the project and contribute to its success or broader research goals. 

Reviewer: Should this also mention the universities 3 priorities, not that I recall. Might be the one university vision, but I'd need to check.

Author: I have added in the introduction: 'To support the *One University* vision and the University research community'. I am not sure how we could include this in the impact statement (if that's where you thought it could go).

Reviewer: Yeah, it's an awkward one but something senior uni people are probably keen to be pushed by anything they're effectively funding.

-**Existing material [max 200 words]:** Briefly explain what existing material you already have: Data, Software, Documentation…

-**Preferred timeline:** what would be the ideal time distribution of the RSE.

-**The application shall be sent to the chair of the panel N. Shephard (see below for contact information) with subject 'Application for RSE call 2024', as PDF no later than the 28th of June at 5pm.**

### Review and selection process

> The panel reviewing the projects will consist of members from the Research Software Engineering team and Data Analytics Service [DAS]. The panel will gather on July where a final selection will be done.

Reviewer: ```suggestion
The panel reviewing the projects will consist of members from the Research Software Engineering team and Data Analytics Service [DAS]. The panel will gather in July where a final selection will be done.
```

The criteria for the selection will be the following:

> -**Fit:** Request fits the objective of the call.

Reviewer 3: Spacing, plus other bullet points.

```suggestion
- **Fit:** Request fits the objective of the call.
```

-**Skills:** Does the RSE team have the required skills? Are these skills available for the duration of the project?
-**Impact:** Potential Impact of the RSE work [For the project, and wider for the community.]

> -**Practicality:** How easy will it be for the RSE to get into the project?

Reviewer: ```suggestion
-**Practicality:** How easy will it be for the RSE to join the project?
```

-**Resource efficiency:** Is the work requested achievable in the offered time?

Selected projects will be advertised on our website (displaying a short abstract of the project).

### Important dates

> -Deadline for proposal submission: June 28th

Reviewer 3: Ditto

```suggestion
- Deadline for proposal submission: June 28th
```

-Notification of Decision: July 22th
-Earliest start of project: September 1st
-Latest start of project: February 3rd

**Important Note:** While the P.I. is requested to provide an ideal starting date, the final starting date might be different. Availability of team members depends on other projects and we will make our best to accommodate the requested starting date. We will therefore ask the P.I.s to understand that if their project is selected, the starting date might be different from what they have requested.

### Contact information

> For inquiries or assistance with the proposal submission process, please contact Neil Shephard (<a href="mailto:n.shephard@sheffield.ac.uk">n.shephard@sheffield.ac.uk</a>, Chair of the selection Panel) and/or Romain Thomas, (<a href="mailto:romain.thomas@sheffield.ac.uk">romain.thomas@sheffield.ac.uk</a>, Head of the RSE group).

Reviewer 3: Doesn't need changing but might save a few keystrokes in the future. I've found you can use Markdown for `mailto:` links too.

```suggestion
For inquiries or assistance with the proposal submission process, please contact Neil Shephard ([n.shephard@sheffield.ac.uk](mailto:n.shephard@sheffield.ac.uk), Chair of the selection Panel) and/or Romain Thomas, ([romain.thomas@sheffield.ac.uk](mailto:romain.thomas@sheffield.ac.uk), Head of the RSE group).
```
